### ISSUE TITLE
Poloniex fee calculation issue

### ISSUE THREAD

**Issue by nmiculinic** [ccxt/ccxt#645]
Per manual:
```
    'fee':      {                // fee info, if available
        'currency': 'BTC',       // which currency the fee is (usually quote)
        'cost': 0.0009,          // the fee amount in that currency
    },
```

while poloniex returns "cost" as multiplicative, not as absolute, for example:
```
CCXT: {'cost': 0.0015, 'currency': 'USDT'}
```

Could you please fix this discrepancy? I see all costs are too exact --> 0.0015/0.0025

**Comment by kroitor** [comment]
> while poloniex returns "cost" as multiplicative, not as absolute, for example

Can you please elaborate on this, namely, which pair it is, which values you expect to see and what you get instead, also, order sizes and prices?

**Comment by nmiculinic** [comment]
Here is the data (after minor processing):
```
              {'amount': 0.02512245,
               'datetime': <redacted>,
               'fee': {'cost': 0.0025, 'currency': 'BTC'},
               'id': '<redacted>',
               'info': {'amount': '0.02512245',
                        'category': 'exchange',
                        'date': '<redacted>',
                        'fee': '0.00250000',
                        'globalTradeID': <redacted>,
                        'orderNumber': '<redacted>',
                        'rate': '0.04930000',
                        'total': '0.00123853',
                        'tradeID': '<redacted>',
                        'type': 'sell'},
               'order': '<redacted>',
               'price': 0.0493,
               'side': 'sell',
               'status': 'closed',
               'symbol': 'ETH/BTC',
               'timestamp': <redacted>,
               'type': 'limit'})])
```

**Comment by nmiculinic** [comment]
It affects all symbols that I'm aware off

**Comment by kroitor** [comment]
This is now fixed in version 1.10.196, thank you for reporting!
